src/decisionengine/framework/taskmanager/ProcessingState.py

```diff
@@ -54,6 +55,7 @@ def __init__(self, state=State.BOOT):
         self._lock = multiprocessing.RLock()
         self._state = multiprocessing.Value('i', allowed_state.value)
         self.logger = logging.getLogger()
+        self.__id = f"{multiprocessing.current_process().name}-{threading.current_thread().name}"
```

Comment:
One too many underscores in the attribute.

Reply:
No wonder it wasn't working ;)